src/tribler-common/tribler_common/patch_import.py

```diff
@@ -8,11 +8,11 @@
 """
 import builtins
 from typing import List
-
-import mock
+from unittest.mock import Mock, patch
```

Comment:
**Issue** : this is a part of the tribler_common package, so this can be used in other parts of code in production.

Therefore it seems to be wrong to change the mock library to `unittest.mock`

**Solution**: keep mock as a dependency as it was before.

Reply:
From https://newbedev.com/python-mock-vs-unittest-mock-patch:

>The mock library has been integrated into the Python standard library from Python version 3.3 on as unittest.mock. They deliver the same functionality.

>Nowadays the (external) mock library is a backport of the version in the standard library. If you are using a recent version of Python and don't have any special version requirement, the version from the standard library should be preferred.

The change is due to `docs-build.sh` failing because of unresolved dependency to `mock`. I was facing the choice of:
* adding `mock` to `requirements.txt`
* switch to `unittest.mock`

I choose the latter option, because it removes the external dependency, which is a backport.

Reply:
`build-docs.sh` tries uses  `doc/requirements.txt`. So, if you don't use the Jenkins job, but instead tries to build it locally, it fails with missing dependencies.

Reply:
Now it is clear, thank you. Don't forget to remove actual `Mock` dependency from

https://github.com/ichorid/tribler/blob/b95ffaa55304c103858fe4dca9724c6433e303ab/src/tribler-common/tribler_common/requirements.txt#L1